tests/test_SklearnPipeline.py

```diff
@@ -5,8 +5,8 @@
 import onnx
 from sklearn import datasets
 from sklearn.linear_model import LogisticRegression
-from sklearn.pipeline import make_pipeline, Pipeline
-from sklearn.preprocessing import OneHotEncoder, StandardScaler
+from sklearn.pipeline import make_pipeline, Pipeline, FeatureUnion
+from sklearn.preprocessing import OneHotEncoder, StandardScaler, MinMaxScaler
```

Comment:
Order alphabetically.

Reply:
done

Reply:
I also meant order the package names alphabetically. For example:
from sklearn.preprocessing import MinMaxScaler, OneHotEncoder, StandardScaler 
instead of
from sklearn.preprocessing import OneHotEncoder, StandardScaler, MinMaxScaler